mergify_engine/worker.py

```diff
@@ -243,11 +243,26 @@ async def run_engine(
         except http.HTTPNotFound:
             # NOTE(sileht): Don't fail if we received even on repo/pull that doesn't exists anymore
             logger.debug("pull request doesn't exists, skipping it")
-            return None
+            return
+
+        except http.HTTPClientSideError as e:
+            if (
+                e.status_code == 422
+                and "The request could not be processed because too many files changed"
+                in e.message
+            ):
+                logger.warning(
+                    "This pull request cannot be evaluated by Mergify", exc_info=True
+                )
+                return
+
+            # NOTE(sileht): Reraise to retry with worker or create a Sentry
+            # issue
```

Comment:
how a Sentry issue can be useful here?

Reply:
Mainly to track and learn what kind of GitHub issue we can have here and not blindly ignore all 4XX.